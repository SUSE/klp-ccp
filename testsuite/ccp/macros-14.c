

#define s hello

#define __str(a) #a
#define str(a) __str(a)

void g(void)
{
	str(s);
}

#undef s

void pu_f(void)
{
	g();
	str(s);
}
