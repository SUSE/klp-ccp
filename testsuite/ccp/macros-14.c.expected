/* klp-ccp: from testsuite/ccp/macros-14.c */
#define s hello

#define __str(a) #a
#define str(a) __str(a)

static void g(void)
{
	str(s);
}

#undef s

void klpp_pu_f(void)
{
	g();
	str(s);
}
